libs/ui-toolkit/src/components/arrows/arrow.tsx

```diff
@@ -1,10 +1,36 @@
+import classNames from 'classnames';
+
 export interface ArrowStyleProps {
   color?: string;
   borderX?: number;
   borderTop?: number;
   borderBottom?: number;
+  up?: boolean;
 }
 
+export const RotatingArrow = ({
+  color = 'green',
```

Comment:
comment (non-blocking): not a huge fan of hardcoded colors on svgs personally, borders as well can work using `currentColor` which just inherit from whatever the parent has as color.

Reply:
updated